components/Storyblok/SbTypeform.tsx

```diff
@@ -62,6 +64,9 @@ export const SbTypeform = ({
   const height = parseInt(embedHeight, 10);
   const resize = autoResize || !height;
   const opacity = transparent ? 0 : 100;
+  const iframeProps = {
+    ariaLabel: embedLabel ?? undefined,
```

Comment:
@sherakama Thank you so much for taking care of this request! Only tiny tweak I would ask for is this line
```suggestion
    ariaLabel: embedLabel ? embedLabel : undefined,
```
The `??` doesn't count empty string as false. Since `embedLabel` is default to `''`, if the field in Storyblok is empty, it will still add `aria-label=""` (or sometimes appear just as `aria-label`) 😄 which I don't think is what you want.

![Screenshot 2024-09-05 at 1 33 42 PM](https://github.com/user-attachments/assets/148381bf-85b7-4e1e-be92-6ed6a0008cb3)

Reply:
Oh shoot yeah. The NULLcoalesce operator should have been `||` as that does truthy.

Reply:
That got me all the time too 😆